new-site/src/docs/elements/components/accordion/usage.mdx

```diff
@@ -27,44 +26,37 @@ import PlaygroundPreview from '../../../../components/Playground'
 - It is recommended to have accordions closed when the page is loaded.
   - If you need to have the accordion initially open, you can use the `initiallyOpen` property to achieve this.
   - Having an accordion initially open could be needed e.g. when the user returns to a page and you want to retain accordions' states.
-Accordions should initially be closed when the page is loaded.
+    Accordions should initially be closed when the page is loaded.
```

Comment:
This looks like it's missing a hyphen (-)?

Reply:
Fixed.
Though I started to think about the usefulness of this when the previous main item says "It is recommended to have accordions closed when the page is loaded"